source/System.cpp

```diff
@@ -268,6 +268,8 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 			trade[value].SetBase(child.Value(valueIndex + 1));
 		else if(key == "object")
 			LoadObject(child, planets);
+		else if(key == "arrival")
+			arrivalDistance = child.Value(valueIndex);
```

Comment:
Should we allow negative values of `arrivalDistance`?

Reply:
What is the impact of negative values? Or I suppose the corollary question would be: Where does `arrivalDistance = 0` leave the player? Is it in the system center? or is is it the normal arrival spot around roughly the habitable zone?

Some speculations that come to mind:
If it is the former, and 0 is the center of the system, would a negative value catapult the player past the center of the system and drop them on the far side? If so, I'd say allow this value, because it make a neat change for certain specific systems or missions (If anyone were doing, say, a mission revolving around investigation of weird hyperspace phenomenon or something.)

What I'm getting at is this: So long as negative values have a noticeable and relatively consistent/predictable impact on ship arrival location, I would say we should allow them.

Reply:
`arrivalDistance = 0` is the default value, which leaves the player arriving at exactly the same spot as where he/she arrives without this PR.

A negative `arivalDistance` value would indeed catapult the player beyond his/her intended destination. I considered blocking negative values, but I actually came to the same conclusion as you; there are probably some content/plugin creators that can do something nice with negative values.

Reply:
Actually, if this gets merged, I'm going to include this in Remnant missions. :) A very long ways down the story tree, but still, fun ideas. : )